### PR TITLE
code works on py2 now

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: tests/test_cdtime_components.py [hunk 1]
```diff
@@ -5,6 +5,8 @@ class CDTIMETest(unittest.TestCase):
     def testCdtimeComponent(self):
         tc = cdtime.comptime(2017)
         value = 4
+        import pdb
+        pdb.set_trace()
```
Comment (doutriaux1) [inline tests/test_cdtime_components.py]: need to remove

File: Src/cdtimemodule.c [hunk 2]
```diff
@@ -92,27 +86,17 @@ static int set_double_to_scalar(double *d, PyObject *value) {
...
 
-#if PY_MAJOR_VERSION <3
     if (PyInt_Check(value)){
-        *lval = PyInt_AsLong(value);
-        return 0;
-    }
-#endif
-    if (PyLong_Check(value)) {
-        *lval = PyLong_AsLong(value);
+        *lval = (long) PyInt_AsLong(value);
```
Comment (doutriaux1) [inline Src/cdtimemodule.c]: does PyInt_* exist in Python 3?

Comment (doutriaux1) [reply Src/cdtimemodule.c]: https://docs.python.org/3/howto/cporting.html
says: 
```
long/int Unification
Python 3 has only one integer type, int(). 
But it actually corresponds to Python 2’s long() type—the int() type used in Python 2 was removed. 
In the C-API, PyInt_* functions are replaced by their PyLong_* equivalents.
```

**Comment by dnadeau4** [comment]
I include py3c.h and the macros takes care of it.

Denis

**Pull request merged**